src/backend/web/app.js

```diff
@@ -22,8 +22,10 @@ const app = express();
 app.use(helmet());
 
 // Enable CORS and preflight checks on all routes
-app.use(cors());
-app.options('*', cors());
```

Comment:
Why remove this?  For preflight checks that need to request options, we'd need this.

Reply:
@humphd 
![image](https://user-images.githubusercontent.com/18711727/78502573-cde92b80-772f-11ea-8534-39b563b23c48.png)

Doesn't this not mean they are doing the same thing?

Reply:
You're right, thanks for helping me understand.